src/main/resources/migration/common/V11__create_licence_tables.sql

```diff
@@ -107,8 +111,8 @@ CREATE INDEX idx_bespoke_condition_licence_id ON bespoke_condition(licence_id);
 
 CREATE TABLE licence_history (
   id serial NOT NULL constraint licence_history_pk PRIMARY KEY,
-  licence_id integer references licence(id),
-  status_code varchar(20) references licence_status(status_code),
+  licence_id integer references licence(id) ON DELETE CASCADE,
```

Comment:
When a licence is discarded the licence_history record also gets deleted, otherwise foreign_key contstraint is violated